.github/workflows/actions_docs.yml

```diff
@@ -23,7 +25,7 @@ jobs:
           make -C docs/ html
       - name: Deploy
         uses: peaceiris/actions-gh-pages@v3
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/actions_docs' }}
```

Comment:
Does this mean that we need to keep the `actions-docs` branch even after merging into `main`?

Reply:
No. I've just changed that line for testing with the current branch, it's reverted back to main again now.